src/scalper.ts

```diff
@@ -685,9 +686,10 @@ export class Scalper {
       (message: SerumVialTradeMessage) => {
         console.log(this.symbol, "Gamma Scalp Filled!", message.size, message.market, message.price, message.timestamp);
         let fillPrice = Number(message.price);
+        gammaScalpCount = gammaScalpCount + 1;
```

Comment:
generally better to not modify an input variable and either make a copy and modify that, or leave it as it was. a linter will probably complain about this.

Reply:
Wanted to confirm that it gets picked up in waitForGamma. Seemed easier to manage just the 1 counter variable for that